src/main/java/com/example/demo/models/CategoryModel.java

```diff
@@ -0,0 +1,33 @@
+package com.example.demo.models;
+
+import com.example.demo.entities.Category;
+
+public class CategoryModel {
+	private int id;
+	private String name;
+	
+	public static CategoryModel fromCAtegoryEntity(Category entity) {
```

Comment:
typo

also, our models should be as simple as possible

model should not concern itself with Entity, it should be other way around

add this logic to entity